CHANGELOG-1.x.md

```diff
@@ -2,6 +2,7 @@
 
 
 ## vN.N.N
+* Add save & close and save & publish dropdown buttons to all nodes.
```

Comment:
@tbone1020 i think your change log needs to be updated to reflect your actual changes.

Reply:
@hunyoboy done.